helm/presets/aws.yaml

```diff
@@ -42,15 +52,21 @@ strimzi:
       limits:
         cpu: 250m
         memory: 512Mi
+
+# Redis subchart
```

Comment:
These type of comments seem like noise.

Reply:
I wanted to call out how values for a subchart are overridden from a parent chart since that is not obvious to everyone.

Reply:
_someone_ who first stood up the hindsight chart didn't have the presto chart properly referenced viz-a-viz overriding subchart values from a top-level `values.yaml` file and we were just getting lucky based on that chart's defaults. 😁

Reply:
I get fixing the organization/structure of the variables.

If you want to have some commentary on the reasoning of the organization/structure, add a comment to the top. Something like:

```
# Subchart configuration is namespaced under the subchart's name
# as a top-level key. For example, the Certificate Manager's subchart is 
# configured under the cert_manager key.
```

To me, that comment is valuable and the many comments of `$keyname subchart` are not.

Reply:
fair enough; i can get behind that.

Reply:
Just pushed a fix; let me know what you think.